src/python/lib/ensembl/compara/orthtools/orthofinder.py

```diff
@@ -0,0 +1,92 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Methods for preparing input files and running OrthoFinder.
+
+This module provides::
+
+    :func:`prepare_input_orthofinder()` which copies input fasta files for OrthoFinder
+            into a specified working directory
+
+    :func:`run_orthofinder()` which runs OrthoFinder
+
```

Comment:
I find this too verbose: the documentation will automatically pull each function's docstring during its generation. However, this is just my personal preference (but bear in mind this would be required to be maintained and updated).

Reply:
I was using `filecmp.py` as a guidance 👼  How much documentation should be written here in general?
(I see the description for `prepare_input_orthofinder` is wrong here (symlinks) but I think that won't be relevant any more.)

Reply:
I'm afraid that is one of the first modules written in Python and we have changed slightly what we aim to include in the module's docstring: brief description, larger description and some usage examples. This is what people tend to do (even the Google docstring example covers it this way) as it is the most informative for users without duplicating much information.

Reply:
All clear except the definition of `brief` and `larger`👼  I'll try with more recent examples from the codebase 😉